R/add_activity_after_death_flag.R

```diff
@@ -125,8 +125,8 @@ add_activity_after_death_flag <- function(
 #'
 #'
 # Read data------------------------------------------------
-process_deaths_lookup <- function(update = latest_update(),
-                                  write_to_disk = TRUE, ...) {
+process_combined_deaths_lookup <- function(update = latest_update(),
+                                           write_to_disk = TRUE, ...) {
```

Comment:
are these meant to be different? I do not want to overwrite changes on the September branch

Reply:
> are these meant to be different? I do not want to overwrite changes on the September branch

Yeah. It's something that Jen and I agreed but she forgot to change. `process_combined_deaths_lookup` make more sense and is more intuitive. This function is used in `run_all_targets.R` and the new name `combined` is already in use.